tests/integration/suite/test_cluster_defaults.py

```diff
@@ -157,7 +157,7 @@ def test_import_initial_conditions(admin_mc, remove_resource):
     cluster = admin_mc.client.create_cluster(name=random_str())
     remove_resource(cluster)
 
-    assert cluster.conditions is None
```

Comment:
are you sure this is never None?  `assert not cluster.conditions` might be safer

Reply:
good catch, fixed